tst/testall.g

```diff
@@ -1,4 +1,4 @@
-#
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Is it possible to have another line starting with # for better readability?

Reply:
We only have a fixed number of header lines (which is 5), as otherwise the header would have to be wrapped in `BEGIN` and `END`, which would look even worse I think. Increasing this number would require to touch every file manually. Thus, this is not easily possible.

Reply:
I see. Then I declare it pretty enough 👍